db/migrate/20200520152604_change_stripe_subscription_schedule_phase_id_to_bigint.rb

```diff
@@ -0,0 +1,17 @@
+class ChangeStripeSubscriptionSchedulePhaseIdToBigint < ActiveRecord::Migration[6.0]
+  def change
+    if postgres?
+      change_column(
+        :stripe_subscription_schedule_phase_plans,
+        :stripe_subscription_schedule_phase_id,
+        "bigint USING stripe_subscription_schedule_phase_id::bigint"
```

Comment:
Why is it necessary with something special for postgres?

Reply:
> **SET DATA TYPE**
> ... A USING clause must be provided if there is no implicit or assignment cast from old to new type.

https://www.postgresql.org/docs/9.1/sql-altertable.html